src/hooks/use-add-site.ts

```diff
@@ -51,22 +50,13 @@ export function useAddSite() {
 	}, [ __, siteWithPathAlreadyExists, siteName, proposedSitePath ] );
 
 	const handleAddSiteClick = useCallback( async () => {
-		setIsAddingSite( true );
 		try {
 			const path = sitePath ? sitePath : proposedSitePath;
 			await createSite( path, siteName ?? '' );
 		} catch ( e ) {
 			Sentry.captureException( e );
-			setError(
-				__(
-					'An error occurred while creating the site. Verify your selected local path is an empty directory or an existing WordPress folder and try again. If this problem persists, please contact support.'
-				)
-			);
-			setIsAddingSite( false );
-			throw e;
```

Comment:
I see this error has been moved to `useSiteDetails` `createSite` 👍

Reply:
Yeah I moved it over as I did not want it to be a part of the add site form anymore and wanted to trigger it when the loading state happens.